package.json

```diff
@@ -95,7 +95,7 @@
     "@angular/core": "^5.0.0",
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
-    "@fortawesome/fontawesome": "^1.2.0",
+    "@fortawesome/fontawesome": "^1.1.4",
```

Comment:
This jumped out at me. Is there a reason for this downgrade? We're actually in the process of releasing stable versions of FA 5.1 which would include the 1.2 of the library.

Reply:
Weird, I didn't touched the package.json file.